insightconnect_plugin_runtime/plugin.py

```diff
@@ -193,7 +193,13 @@ def envelope(message_type, input_message, log, success, output, error_message, s
         if success:
             output_message["output"] = output
         else:
-            output_message["error"] = error_message
+            output_message["error"] = str(ex)
+            if isinstance(ex, ConnectionTestException) or isinstance(ex, PluginException):
```

Comment:
```suggestion
            if isinstance(ex, ConnectionTestException) or isinstance(ex, PluginException):
```
Should only need ConnectionTestException, I believe, since PluginException inherits from CTE